docs/ALGOLIA-SEARCH.md

```diff
@@ -0,0 +1,65 @@
+<h1 align="center">Algolia Search</h1>
+
+
+NexT provides search plugin for index your hexo website content. To use this feature, make sure that the version of NexT you are using is after the v5.1.0 release. What you should note here is that only turn on `enable` of `algolia_search` in `next/_config.yml` cannot let you use the algolia search correctly, you need to install corresponding [Hexo Algolia](https://github.com/oncletom/hexo-algolia) plugin to seach your website with Algolia. Follow the steps described below to complete the installation of Algolia search.
+
+1. Register at [Algolia](https://www.algolia.com/), you can log in directly using GitHub or Google Account. Upon Customer’s initial sign-up for an Account, Customer will have a free, fourteen (14) day evaluation period (the “Evaluation Period”) for the Algolia Services commencing on the Effective Date, subject to the limitations on Algolia’s website. After that, Algolia offers a free, branded version for up to 10k records and 100k operations per month.
+
+1. If a tutorial pops up, you can skip it. Go straight to create an `Index` which will be used later.
+
+    ![](http://theme-next.iissnan.com/uploads/algolia/algolia-step-2.png)
+
+1. Algolia requires users to upload their search index data either manually or via provided APIs. Install and configure [Hexo Algolia](https://github.com/oncletom/hexo-algolia) in your Hexo directory. This plugin will index your site and upload selected data to Algolia.
+
+    ```
+    cd hexo
+    npm install --save hexo-algolia
+    ```
+
+1. Go to API Keys and find your credentials. You will need the `Application ID`, the `Search-only API key` and the `Admin API key` in the following sections. The `Admin API key` need to keep confidential.
+
+    ![          ](http://theme-next.iissnan.com/uploads/algolia/algolia-step-3.png)
```

Comment:
Why space?

Reply:
oh, it's a insertion error. 
i will fix it.